sources/shared/Stride.NuGetResolver/NuGetAssemblyResolver.cs

```diff
@@ -145,13 +147,26 @@ internal static void SetupNuGet(string packageName, string packageVersion)
 
                             // Only allow this specific version
                             var versionRange = new VersionRange(new NuGetVersion(packageVersion), true, new NuGetVersion(packageVersion), true);
-                            var (request, result) = RestoreHelper.Restore(logger, nugetFramework, "win", packageName, versionRange);
+                            var (request, result) = RestoreHelper.Restore(logger, nugetFramework, RuntimeInformation.RuntimeIdentifier, packageName, versionRange);
                             if (!result.Success)
                             {
                                 throw new InvalidOperationException($"Could not restore NuGet packages");
                             }
 
                             assemblies = RestoreHelper.ListAssemblies(result.LockFile);
+
+                            // Register the native libraries
```

Comment:
Can this be extracted into a method? And maybe add a new line after each if for readability.

Reply:
Done